CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.5.1 (2020-04-14)
+
+### Added
+Pin gevent to version 1.4.0
+
+
```

Comment:
A bit off topic, but why there is no mention on 0.5.0? This seems like a big update that was omitted.

Reply:
yes indeed I agree that this was forgotten (ping @nottrobin  :wink: )
I think I'll add it in another PR with another change that I'll bring in there.

Reply:
Oops sorry